taskdb/src/redis_db.rs

```diff
@@ -140,10 +153,18 @@ impl RedisTaskDb {
     fn new(config: RedisConfig) -> RedisDbResult<Self> {
         let url = config.url.clone();
         let client = Client::open(url).map_err(RedisDbError::RedisDb)?;
-        let conn = client.get_connection().map_err(RedisDbError::RedisDb)?;
-        Ok(RedisTaskDb { conn, config })
+        Ok(RedisTaskDb { client, config })
     }
 
+    fn get_conn(&mut self) -> Result<Connection, RedisError> {
+        match self.client.get_connection() {
+            Ok(conn) => Ok(conn),
+            Err(_) => {
```

Comment:
Suggest adding a warning log to show the connection error.

Reply:
Done!